src/Symfony/Component/DependencyInjection/Attribute/TaggedItem.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Attribute;
+
+/**
+ * An attribute to tell under which index a class should be found in tagged iterators/locators.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
+class TaggedItem
+{
+    public function __construct(
+        public string $index,
+        public ?int $priority = null,
+        public ?string $type = null,
+    ) {
```

Comment:
Could we describe these? Especially the `$type`... it would be ok with me if they were described above the class. I like documentation :p